src/components/boardmember/style.less

```diff
@@ -0,0 +1,36 @@
+@card-width: 120px;
+
+.board-member-card {
+    display: inline-flex;
+    flex-direction: column;
+    -ms-flex-direction: column;
+    background-color: #F0F0F0;
+    padding: 10px 0px;
+    margin: 10px;
+    border-radius: 20px;
+    box-shadow: 3px 4px 4px rgba(0, 0, 0, 0.25);
+}
+
+.board-member-photo {
+    width: @card-width;
```

Comment:
Is there a way we can increase the card size and possibly shrink it down using a media query on mobile? This is what the width looks like on the Figma 
![image](https://user-images.githubusercontent.com/8893880/80935468-9945af80-8d81-11ea-9c30-4e51857a345b.png)

Reply:
Looks like the desktop vs mobile cards have differing padding values so I added that into the CSS